packages/fxa-payments-server/src/routes/Subscriptions/Cancel/CancelSubscriptionPanel.tsx

```diff
@@ -0,0 +1,196 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I moved this component from SubscriptionItem into its own file.  It's a little more consistent since Reactivate has its own dir and all.  It's also a little easier to find this file or the tests by having 'cancel' in the filenames.